e2e/ios/E2EOktaReactNativeUITests/BrowserLoginTests.swift

```diff
@@ -50,11 +50,7 @@ final class BrowserLoginTests: LoginTests {
     
     let passwordField = webView.secureTextFields.allElementsBoundByIndex.first(where: { $0.frame.width >= usernameField.frame.width })!
     passwordField.tap()
-
-    // `typeText` works weird. Sometimes it doesn't type correct text.
-    UIPasteboard.general.string = password
-    passwordField.doubleTap()
-    app.menuItems["Paste"].tap()
+    passwordField.typeText(password)
```

Comment:
app.menuItems["Paste"].tap() wasn't working. I replaced it with typeText instead, and it seemed to work fine

Reply:
Sounds good. Some versions of Xcode/the iOS Simulator have trouble with reliably typing text, but I think it improved recently.